docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -202,3 +134,143 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
+```
+
+You may notice roles do not define a name. As opposed to the xref:api:access.adoc#AccessControl[`AccessControl`] case, roles aren't hardcoded in the contract as `bytes32` values but identified as numeric values. However, role labeling allows tooling discoverability for easier role exploration and it can be achieved using the xref:api:access.adoc#AccessManager-labelRole-uint64-string-[`labelRole`] function.
+
+```javascript
+await manager.labelRole(MINTER, "MINTER");
+```
+
+It's recommended to keep only a single admin address secured under a multisig or governance layer. To achieve this, it is possible for the initial admin to set up all the required permissions, targets, and functions, assign a new admin, and finally renounce its admin role.
+
+For improved incident response coordination, the manager includes a mode where administrators can completely close a target contract. When closed, all calls to restricted functions in a target contract will revert.
+
+Closing and opening contracts don't alter any of their settings, neither permissions nor delays. Particularly, the roles required for calling specific functions are not modified.
+
+This mode is useful for incident response operations that require temporarily shutting down a contract in order to evaluate emergencies and reconfigure permissions.
+
+```javascript
+// Token's `restricted` functions closed
+await manager.setTargetClosed(myToken.address, true);
+
+// Token's `restricted` functions open
+await manager.setTargetClosed(myToken.address, false);
+```
+
+WARNING: Even if a manager defines permissions for a contract function, these won't be applied if the managed contract instance is not using the xref:api:access.adoc#AccessManageddf-restricted--[`restricted`] modifier for that function, or if its manager is a different one.
+
+=== Role Admins and Guardians
+
+An important aspect of the AccessControl contract is that roles aren't granted nor revoked by role members. Instead, it relies on the concept of a role admin for granting and revoking.
+
+In the case of the AccessMananger, the same rule applies and only the role's admins are able to call xref:api:access.adoc#AccessManager-grantRole-uint64-address-uint32-[grant] and xref:api:access.adoc#AccessManager-revokeRole-uint64-address-[revoke] functions. Note that calling these functions will be subject to the execution delay that the executing role admin has.
+
+Additionally, the access manager stores a _guardian_ as an extra protection for each role. This guardian has the ability to cancel operations that have been scheduled by any role member with an execution delay. Consider that a role will have its initial admin and guardian default to the `ADMIN_ROLE` (`0`).
+
+IMPORTANT: Be careful with the members of `ADMIN_ROLE`, since it acts as the default admin and guardian for every role. A misbehaved guardian can cancel operations at will, affecting the manager's operation.
```

Comment:
```suggestion
IMPORTANT: Be careful with the members of `ADMIN_ROLE`, since it acts as the default admin and guardian for every role. A misbehaved guardian can cancel operations at will, affecting the `AccessManager`'s operation.
```

Reply:
What do we mean by "a misbehaved guardian"?

Reply:
A guardian is implicitly able to DoS the role is guarding. By "misbehaved guardian" we're referring to an address granted with a role (guardian of another) that cancels every operation from its guarded role members.